mailbox/store/src/main/java/org/apache/james/mailbox/store/event/MailboxUpdateParentAsLookupListener.java

```diff
@@ -0,0 +1,103 @@
+/****************************************************************
```

Comment:
This class lack test!!!!

I want a 100% code coverage and a 100% fonctional coverage of it!